src/validations/range.js

```diff
@@ -28,7 +28,7 @@ export default (data, field, message, [min, max], get) => {
 
     const fieldValue = get(data, field)
 
-    if (!skippable(fieldValue) && !between(fieldValue, min, max)) {
+    if (!skippable(fieldValue) && !between(fieldValue, min, max, true)) {
```

Comment:
@danilopolani This would break the previous use-case. How about adding support for additional arguments too and fallback to the previous behavior which is not inclusive?

```js
import boolean from '../raw/boolean'
...
// default is false to avoid breaking changes
export default (data, field, message, [min, max, inclusive = false], get) => {

if (!boolean(inclusive)) {
  return new Error('range:inclusive value should be one of [true, false, 0, 1]')
}

if (!skippable(fieldValue) && !between(fieldValue, min, max, inclusive)) {
...
```